train_thai.py

```diff
@@ -33,4 +33,4 @@
                                                                 word_segmenter.hunits))
 # word_segmenter.save_model()
 word_segmenter.test_model_line_by_line(verbose=True, fast=True)
-# '''
+# '''
```

Comment:
I don't understand this line (maybe it is me using GitHub after years). It seems the removed and added lines are identical?